examples/document/basic-example.html

```diff
@@ -3007,7 +3007,7 @@ <h2 id="_conclusion">Conclusion</h2>
   }
 })()
 </script>
-<script type="text/javascript" src="file:///home/guillaume/workspace/opensource/asciidoctor/asciidoctor-pdf.js/node_modules/mathjax/es5/tex-chtml-full.js"></script>
+<script type="text/javascript" src="file:///C:/workspace/asciidoc/asciidoctor-pdf.js/node_modules/mathjax/es5/tex-chtml-full.js"></script>
```

Comment:
Should we embed this file in the HTML page (like the other "built-in" resources)?

Reply:
I think this is unrelated. Maybe switching to a relative path would be nicer here.

Reply:
You're right 👍